src/map/skill.cpp

```diff
@@ -7759,6 +7832,15 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		clif_skill_nodamage(src,bl,skill_id,skill_lv,
```

Comment:
NPC_DAMAGE_HEAL is missing in skill_castend_nodamage_id
```suggestion
	case NPC_DAMAGE_HEAL:
		clif_skill_nodamage(src,bl,skill_id,skill_lv,
```

Reply:
Thanks